Documentation/concepts/kubernetes/compatibility-table.rst

```diff
@@ -146,7 +146,9 @@
 +-----------------+----------------+
 | v1.12.16        | 1.25.7         |
 +-----------------+----------------+
+| v1.12.17        | 1.25.7         |
++-----------------+----------------+
 | v1.12           | 1.25.7         |
 +-----------------+----------------+
-| latest / master | 1.26.7         |
+| latest / master | 1.26.9         |
```

Comment:
As discussed [here](https://github.com/cilium/cilium/pull/30307#discussion_r1457241411), results from an outdated script but we can address it as a follow-up.